index.html

```diff
@@ -180,8 +180,8 @@ <h1 style="margin-bottom: 7px;">Text to Handwriting</h1>
       <h2>✨ How to add your own handwriting?</h2>
       <ul>
         <li>To use your handwriting, you will have to generate font of your handwriting.</li>
-        <li>There are websites like <a href="https://www.calligraphr.com/en/">Calligraphr</a> that let you do that.</li>
-        <li>Once you get .ttf file of your handwriting, upload it from upload button and boom! you will have your handwriting in the image.</li>
+        <li>There are websites like <a target="_blank" href="https://www.calligraphr.com/en/">Calligraphr</a> that let you do that.</li>
```

Comment:
Using `target="_blank"` is actually a bad practice since it changes the default behavior of web. Here's an article on that: https://css-tricks.com/use-target_blank/ 

If a user wants to open it a new tab, they can hold CTRL and click link which is what a lot of times users expect.